tests/requirements-relaxed.txt

```diff
@@ -125,7 +125,7 @@ sphinx==7.1.2
     #   sphinxcontrib-serializinghtml
 sphinx-ansible-theme==0.10.2
     # via -r tests/requirements-relaxed.in
-sphinx-notfound-page==0.8.3
+sphinx-notfound-page==1.0.0
```

Comment:
This might allow to bump (resp. unrestrict) the Sphinx version.

Reply:
> This might allow to bump (resp. unrestrict) the Sphinx version.

See https://github.com/ansible/ansible-documentation/pull/365.